backend/events/migrations/0001_initial.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 2.1.7 on 2019-03-19 00:21
```

Comment:
Can we squash these migrations down into one? No reason to have 4 of them in a single commit.

Reply:
Is there a command that we can run to combine them?

Reply:
Just migrate down, delete them all, then re-run makemigrations.

Reply:
squashed